lib/assets/stylesheets/modules/_search.scss

```diff
@@ -16,15 +16,16 @@
       overflow: hidden;
     }
     .form-control {
-      @include box-sizing(border-box);
-      @include core-19;
+      @include govuk-font($size: 19);
+
+      box-sizing: border-box;
       width: 100%;
       padding: 5px 4px 2px;
-      border: 2px solid $text-colour;
+      border: 2px solid  $govuk-text-colour;
       appearance: none;
 
       &:focus {
-        outline: 3px solid $focus-colour;
+        outline: 3px solid $govuk-focus-colour;
```

Comment:
The border on the search box should get thicker on focus, like other text inputs in Frontend. I wonder if it might make more sense just to use a `govuk-input` and remove the custom styling?